appium/tests/specs/mock/inbox/CheckArchiveThreadTooAggressive.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { MockApi } from 'api-mocks/mock';
+import { MockApiConfig } from 'api-mocks/mock-config';
+import { MockUserList } from 'api-mocks/mock-data';
+import {
+  SplashScreen,
+  SetupKeyScreen,
+  MailFolderScreen,
+  EmailScreen
+} from '../../../screenobjects/all-screens';
+
+describe('INBOX: ', () => {
+
+  it('check "archive thread" too aggressive', async () => {
+    const mockApi = new MockApi();
+
+    const testMessage = 'Test "archive thread" too aggressive';
+    const addedMessageSubject = 'Test "archive thread" too aggressive new message';
+    mockApi.fesConfig = MockApiConfig.defaultEnterpriseFesConfiguration;
+    mockApi.ekmConfig = MockApiConfig.defaultEnterpriseEkmConfiguration;
+    mockApi.addGoogleAccount('e2e.enterprise.test@flowcrypt.com', {
+      messages: [testMessage],
+    });
+    mockApi.attesterConfig = {
+      servedPubkeys: {
+        [MockUserList.e2e.email]: MockUserList.e2e.pub!
+      }
+    };
+
+    await mockApi.withMockedApis(async () => {
+      await SplashScreen.mockLogin();
+      await SetupKeyScreen.setPassPhrase();
+      await MailFolderScreen.checkInboxScreen();
+
+      await browser.pause(1000);
+      mockApi.addGoogleMessage('e2e.enterprise.test@flowcrypt.com', addedMessageSubject);
+      await MailFolderScreen.refreshMailList();
+      await browser.pause(10000000);
```

Comment:
@sosnovsky `addGoogleMessage` doesn't seem to work as expected (or am i doing something wrong?)
It is showing 2 threads if I call this method.
Could you check here?

![image](https://user-images.githubusercontent.com/77344191/181517099-a0292164-bb7d-4444-b344-2f3aa08f1f37.png)

Reply:
@ioanmo226 `message-export-18244f5e83a4a7aa.json` has different `threadId` than previous messages. It should be `18244f525d7e0023` instead of `18244f5e83a4a7aa`

Reply:
Aha, got it. Thanks